packages/elements-core/package.json

```diff
@@ -14,8 +14,16 @@
     "url": "https://github.com/stoplightio/elements"
   },
   "license": "Apache-2.0",
-  "main": "index.js",
-  "exports": "./index.js",
+  "type": "commonjs",
```

Comment:
this is needed because the packages in node_modules are symlinks linking to each workspace package.
Due to that, `require('./dist/index.js')` or its ESM equivalent do not work, since the file does not exist.
These are 1:1 mirrors of `dist/package.json` of each package, with that difference that each path is prefixed with `./dist`.
This doesn't affect the actual `dist/package.json` as `stoplight/scripts` overrides type, main, module, and other fields, so we'll still end up having correct path.